force-app/main/default/classes/SFDCStopTeamController.cls

```diff
@@ -98,7 +98,14 @@ public class SFDCStopTeamController {
                 'https://trailblazer.me/id/akohale'
             )
         );
-
+          sfdcstopContributors.add(git checkout -b
+            new Contributor(
```

Comment:
Hi @nsandeep078 can you please remove this git checkout -b from the file ? Everything else is fine and I'll merge it. Thanks!